ptbcontrib/username_to_chat_api/usernametochatapi.py

```diff
@@ -20,63 +20,51 @@
 This module contains a class that, once initiated, works as a shortcut to the UsernameToChatAPI
 and puts the response in a Chat object, as well as puts the error to the fitting TelegramErrors.
 """
+from httpx import AsyncClient
 from telegram import Bot, Chat, error
 
-try:
-    from telegram.vendor.ptb_urllib3 import urllib3
-except ImportError:  # pragma: no cover
-    import urllib3
 
-try:
-    import ujson as json
-except ImportError:  # pragma: no cover
-    import json  # type: ignore[no-redef]
-
-
-class UsernameToChatAPI:  # pylint: disable=too-few-public-methods
+class UsernameToChatAPI:
     """
-    This class stores the URL and api key for a UsernameToChatAPI instance, and then provides a
-    method to get the Chat object for an username.
-
-     Args:
-         api_url (:obj:`str`): The base URL (speak: domain) to the API.
-         api_key (:obj:`str`): The key, necessary to access the API.
-         bot (:obj:`telegram.Bot`): An initiated bot object, to create the Chat object.
-         pool_manager (:obj:`urllib3.PoolManager`, optional): Pre initialized
-             :obj:`urllib3.PoolManager`.
+    Args:
+        api_url (:obj:`str`): URL to the API instance.
+        api_key (:obj:`str`): Key for the API.
+        bot (:class:`telegram.Bot`): Bot instance, used to create the Chat object.
+        initialized_httpx_client (:class:`httpx.AsyncClient`, optional):
+         Initialized httpx AsyncClient. Will be created otherwise
     """
 
     def __init__(
-        self, api_url: str, api_key: str, bot: Bot, pool_manager: urllib3.PoolManager = None
+        self, api_url: str, api_key: str, bot: Bot, initialized_httpx_client: AsyncClient = None
```

Comment:
```suggestion
        self, api_url: str, api_key: str, bot: Bot, httpx_client: AsyncClient = None
```

Putting that into the docstring should be enough imo.